frame/bounties/src/lib.rs

```diff
@@ -131,11 +136,31 @@ pub trait Config: frame_system::Config + pallet_treasury::Config {
 
 	/// Weight information for extrinsics in this pallet.
 	type WeightInfo: WeightInfo;
+
+	/// Maximum number of subbounty that can be added to active bounty.
+	type MaxSubBountyCount: Get<u32>;
 }
 
 /// An index of a bounty. Just a `u32`.
 pub type BountyIndex = u32;
 
+/// A Subbounty proposal.
+#[derive(Encode, Decode, Clone, PartialEq, Eq, RuntimeDebug)]
+pub struct SubBounty<AccountId, Balance, BlockNumber> {
+	/// The account proposing it.
+	proposer: AccountId,
+	/// The (total) amount that should be paid if the bounty is rewarded.
+	value: Balance,
+	/// The curator fee. Included in value.
+	fee: Balance,
+	/// The deposit of curator.
+	curator_deposit: Balance,
+	/// The amount held on deposit (reserved) for making this proposal.
+	bond: Balance,
```

Comment:
why need this?

Reply:
"value: Balance" :: State variable for managing
payout + fee, to be rewarded to sub-curator & executor.